packages/wallets/src/features/cashier/modules/WithdrawalVerification/WithdrawalVerificationRequest/WithdrawalVerificationRequest.tsx

```diff
@@ -9,24 +11,25 @@ type TProps = {
 const WithdrawalVerificationRequest: React.FC<TProps> = ({ sendEmail }) => {
     return (
         <div className='wallets-withdrawal-verification-request'>
-            <div className='wallets-withdrawal-verification-request-icon'>
-                <EmailVerification />
-            </div>
-            <div className='wallets-withdrawal-verification-request__messages'>
-                <p className='wallets-withdrawal-verification-request__title'>
-                    Please help us verify your withdrawal request.
-                </p>
-                <p className='wallets-withdrawal-verification-request__description'>
-                    <span>
+            <WalletsActionScreen
+                actionText='Send email'
+                description={
+                    <WalletText>
```

Comment:
`description` is already wrapped inside `WalletText` in `WalletsActionScreen` component.

Now I'm thinking, should that be the case? This is out of scope, but `WalletsActionScreen` maybe doesn't need to put `description` inside  `WalletText`. Instead can just render `description` as it is. The consumer of `WalletsActionScreen`, `WithdrawalVerificationRequest` here, can have the freedom to compose their own description 🤔 
Something like this for example, can avoid `br` as well.
```
description = {
<React.Fragment>
      <WalletText lineHeight='xl(?)'> Click the button below and we’ll send you an email with a link. Click that link to verify your withdrawal request.</WalletText>
      <WalletText>This is to protect your account from unauthorised withdrawals.</WalletText>
</React.Fragment>
}
```

Reply:
I see, the issue was that I couldn't add `br` without wrapping the text sections in any tag. Sure, I'll update the description with this solution thanks 👍